pkg/config/config_windows.go

```diff
@@ -6,5 +6,5 @@ func GetPlatformDefaultConfig() []byte {
 		`os:
   openCommand: 'cmd /c "start "" {{filename}}"'
   openLinkCommand: 'cmd /c "start "" {{link}}"'
-  copyToClipboardCommand: 'cmd \c "echo -n {{str}} > /dev/clipboard"'`)
+  copyToClipboardCommand: 'cmd /c "echo -n {{str}} > /dev/clipboard"'`)
```

Comment:
Just made an example program for copying something on windows,  
It was a bit more complicated but in the end it's just calling some dll functions:  
```go
package main

import (
	"fmt"
	"syscall"
	"unsafe"
)

var (
	user32           = syscall.NewLazyDLL("user32.dll")
	openClipboard    = user32.NewProc("OpenClipboard")
	closeClipboard   = user32.NewProc("CloseClipboard")
	setClipboardData = user32.NewProc("SetClipboardData")
	emptyClipboard   = user32.NewProc("EmptyClipboard")

	kernel32     = syscall.NewLazyDLL("kernel32")
	globalAlloc  = kernel32.NewProc("GlobalAlloc")
	globalFree   = kernel32.NewProc("GlobalFree")
	globalLock   = kernel32.NewProc("GlobalLock")
	globalUnlock = kernel32.NewProc("GlobalUnlock")
	lstrcpy      = kernel32.NewProc("lstrcpyW")
)

func main() {
	res, _, err := openClipboard.Call()
	if res == 0 {
		fmt.Println("err:", err)
		return
	}
	defer closeClipboard.Call()
	emptyClipboard.Call(0)

	// This will get placed in the clipboard
	testStr := "testje lmao"

	testStrUTF16 := syscall.StringToUTF16(testStr)
	testStrGlobalAlloc, _, err := globalAlloc.Call(0x0002, uintptr(len(testStrUTF16)*int(unsafe.Sizeof(testStrUTF16[0]))))
	if testStrGlobalAlloc == 0 {
		fmt.Println("err:", err)
		return
	}
	defer func() {
		if testStrGlobalAlloc != 0 {
			globalFree.Call(testStrGlobalAlloc)
		}
	}()

	res, _, err = globalLock.Call(testStrGlobalAlloc)
	if res == 0 {
		fmt.Println("err:", err)
		return
	}

	res, _, err = lstrcpy.Call(res, uintptr(unsafe.Pointer(&testStrUTF16[0])))
	if res == 0 {
		fmt.Println("err:", err)
		return
	}

	res, _, err = globalUnlock.Call(testStrGlobalAlloc)
	if res == 0 {
		if err.(syscall.Errno) != 0 {
			fmt.Println("err:", err)
			return
		}
	}

	res, _, err = setClipboardData.Call(
		uintptr(13),
		testStrGlobalAlloc,
	)
	if res == 0 {
		fmt.Println("err:", err)
		return
	}
	testStrGlobalAlloc = 0
}
```